src/Raven.Pal/src/posix/linuxonly.c

```diff
@@ -108,5 +108,14 @@ rvn_test_storage_durability(
     return rc;
 }
 
+PRIVATE int32_t
+rvn_pipe_for_child(int descriptors[])
```

Comment:
```suggestion
rvn_pipe_for_child(int descriptors[2])
```

Or `int * descriptors` ?